frontend/lib/data/user_repository.py

```diff
@@ -32,3 +32,40 @@ def list(self):
             response = self.table.scan(ExclusiveStartKey=response["LastEvaluatedKey"])
             data.extend(response["Items"])
         return data
+
+
+class RemoteUserRepository:
+    """
+    This will handle all the CRUD actions for users
+    in the AWS cognito user pool
+    """
+
+    def __init__(self, endpoint_url: str, region_name: str, identity_pool_id: str):
+        self.identity_pool_id = identity_pool_id
+        self.provider_name = (
```

Comment:
provider_name sounds like a string instead of tuple, can we rename this to something else?

Reply:
The brackets here are used for wrapping the string in a separate line. It would normally exceed the line length limit. the type of this is actually `str` not `tuple`.  
```
>>> type((
...     "hello there"
... ))
<class 'str'>
```